doc/user_guide/docker/docker_usage.md

```diff
@@ -174,3 +174,14 @@ The following command enables you to log in as the user `root` to the specified
 ```shell
 docker exec --user root -it ${CONTAINER_ID} bash
 ```
+
+## Installing Additional Dependencies
```

Comment:
the magic is not specific to docker

Reply:
Moved to general User Guide.
Left comment about discarding additional dependencies on `docker rm` in User Guide for Docker Edition.